engines/suppliers/app/controllers/suppliers/api/v1/orders_frequencies_controller.rb

```diff
@@ -0,0 +1,81 @@
+module Suppliers
+  module Api
+    module V1
+      class OrdersFrequenciesController < ApplicationController
+        before_action :authenticate
+        before_action :load_orders_frequency, only: [:show, :update]
+        before_action :load_group, only: [:index, :create]
+
+        # GET /api/v1/orders_frequencies?group_id=
+        #
+        # You must pass `group_id` param
+        #
+        def index
+          orders_frequencies = OrdersFrequency.where(group_id: @group.id)
+
+          render json: OrdersFrequenciesSerializer.new(orders_frequencies)
+        end
+
+        # GET /api/v1/orders_frequencies/:id
+        #
+        def show
+          render json: OrdersFrequencySerializer.new(@orders_frequency)
+        end
+
+        # POST /api/v1/orders_frequencies
+        #
+        def create
+          orders_frequency = OrdersFrequency.new(
+            group: @group,
+            ical: orders_frequency_params[:ical],
+            frequency_type: orders_frequency_params[:frequency_type]
+          )
```

Comment:
if at this point we only have the attributes permitted down below, I think it's enough with `OrdersFrequency.new(orders_frequency_params)`